src/Command/AddCommand.php

```diff
@@ -42,7 +43,7 @@ public function getOptionParser(): ConsoleOptionParser {
 	/**
 	 * @param \Cake\Console\Arguments $args Arguments
 	 * @param \Cake\Console\ConsoleIo $io ConsoleIo
-	 * @return int|null|void
+	 * @return null|void
```

Comment:
Well, thats a bit weird ;)
I wonder if we could just set it void then? If we dont plan on returning any actual value

Reply:
Actually, looking at others it seems one could also consolidate this to

    int|null
    
as return type and be done with it :)

Reply:
I would agree if there would be any kind of `return $something;` in that method but there is only a simple `return` in there.

Reply:
In that case the change doesnt make sense though, as future changes to it could return an int code.
And we document the possible types, not actual.

Reply:
right, didn't think about that 🤔